TSPL.docc/LanguageGuide/Concurrency.md

```diff
@@ -1584,10 +1587,6 @@ See also this PR that adds Sendable conformance to FileDescriptor:
 https://github.com/apple/swift-system/pull/112
 -->
 
-For more information about
-suppressing an implicit conformance to a protocol,
-see <doc:Protocols#Implicit-Conformance-to-a-Protocol>.
-
```

Comment:
We might want to keep some form of cross-reference here, since the Protocols chapter contrasts the `~Protocol` syntax with this syntax.  Maybe add:

```
You can also use an unavailable conformance
to suppress implicit conformance to a protocol,
as discussed in <doc:Protocols#Implicit-Conformance-to-a-Protocol>.
```